### PR TITLE
Specify the grid transformation in surveys

### PR DESCRIPTION
**Pull request by audunska**
Question: Should we bump the iteration? Doesn't seem like this has happened too much...

### CODE REVIEW

**Comment by audunska** [comment]
I don't suppose we need to support this in v0 too?

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 1]
```diff
@@ -19,6 +19,37 @@ message LineSelect {
...
+}
+
+// Specify the grid by giving the coordinates of three or more corners
+message TraceCorners {
+    repeated DoubleTraceCoordinates corners = 1;
+}
+
+// Specify the affine transformation between line indices and coordinates
+message GridSpecifier {
+    oneof specifier {
```
Comment (amorken) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:47]: https://community.opengroup.org/osdu/data/data-definitions/-/blob/master/E-R/abstract/AbstractBinGrid.1.0.0.md - could we steal some naming, etc, from here?

Comment (audunska) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:47]: Hmm, it feels a bit of a clumsy way to specify...

**Comment by audunska** [comment]
Should we return the affine_transformation in the Survey proto?

**Review by amorken — APPROVED** [review]

**Pull request merged**